modules/virt-example-host-vrf.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+[id="virt-example-host-vrf_{context}"]
+= Example: Network interface with a VRF instance node network configuration policy
+
+Associate a Virtual Routing and Forwarding (VRF) instance with a network interface by applying a `NodeNetworkConfigurationPolicy` custom resource (CR). 
+
+:FeatureName: Associating a VRF instance with a network interface
+include::snippets/technology-preview.adoc[]
+
+By associating a VRF instance with a network interface, you can support traffic isolation, independent routing decisions and the logical separation of network resources. 
+
+In a bare-metal environment, you can announce load balancer services through interfaces belonging to a VRF instance by using MetalLB. For more information, see the _Additional resources_ section.
+
+The following YAML file is an example of associating a VRF instance to a network interface.
+It includes samples values that you must replace with your own information.
```

Comment:
> It includes samples values that you must replace with your own information.

When I first read this and then noticed no `<replaceable_value>` style markup in the code block, it gave me pause but thought it was probably fine since it's an example/reference. But in later sections (like "Configuring symmetric routing using VRF-based networks with MetalLB"), there were code blocks within procedure steps where it could be maybe not clear what should be replaced (due to lack of `<replaceable_value>` styling). And then in other sections, I see that the `<replaceable_value>` style is used.

I guess my comment is just consider maybe syncing up on the styling, or make it more explicitly clear in callouts when something is just an example value that should be replaced, if it doesn't have the `<>`s.

FWIW, I like the "Egress service custom resource" styling (at least for the more reference-style modules) where it uses `<replaceable_values>` and callouts in a first code block, then has a second block with just the example values plugged in.

Reply:
I take your point on this one. The difficulty is that I'm touching various sections and I am trying to match the style of the different sections so it doesn't stand out too much from an assembly. It's a bit of a balance between getting new content up to muster, but then knitting it into existing content. Hence the differing approaches. In these instances, I wish the style guide would be more prescriptive so we would have greater consistency consistent across the docs...